src/GoLoadUp.php

```diff
@@ -96,6 +96,35 @@ public function validateCartRequirement(): void
                 IneligibleWhiteGloveItemsFoundInCart::class,
                 $requirement?->errorMessage()
             );
+
+        };
+
+        $this->validateCartRequirementQuantity();
```

Comment:
Instead of looping through the entire cart twice and doing almost the same prep, is it possible to do the quantity check in the same loop? You can still extract it out to a separate function that just accepts the `$cartItem`.

Reply:
Great suggestion @tonning, I made a few changes to the code, can you check it out again. Thank youuuuuu.

Reply:
@tonning were you able to check this out again?

Reply:
@tonning Have you been able to check this out again?